File: CarRentalApp/CarRentalApp/Models/Car.cs
﻿using System.Web;
using System.ComponentModel.DataAnnotations;

namespace CarRentalApp.Models
{
    public class Car 
    {
            [Key]
            public  string LicensePlate { get; set; }
            public  string Brand { get; set; }
            public  string Model { get; set; }
            public  short NrOfDoors { get; set; }
            public  short NrOfSeats { get; set; }
            public  bool HasAutomaticGearbox { get; set; }
            public  float PricePerDay { get; set; }
            public Images[] ImageList { get; set; }
            public string ImgCars { get; set; }  
            public string CurrentCity { get; set; }
    }
}

File: CarRentalApp/CarRentalApp/ClientApp/src/app/models/car.ts
import { Images } from './Images';

export interface Car {
  licensePlate: string,
  brand: string,
  model: string,
  nrOfDoors: number,
  nrOfSeats: number,
  hasAutomaticGearbox: boolean,
  pricePerDay: number,
  ImageList: Images[],

}
